riscv-asm.md

```diff
@@ -183,6 +183,41 @@ Directive    | Arguments                      | Description
 .balign      | b,[pad_val=0]                  | byte align
 .zero        | integer                        | zero bytes
 .variant_cc  | symbol_name                    | annotate the symbol with variant calling convention
+.attribute   | name, value                    | RISC-V object attributes, more detailed description see [.attribute](#.attribute).
+
+## <a name=.attribute></a> `.attribute`
+
+`.attribute` directive is used for record information about an object
+file/binary that a linker or runtime loader needs to check compatibility.
+
+More information can refer [attribute section in RISC-V psABI](https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-elf.adoc#attributes).
+
+`.attribute` take two argument, first argument of `.attribute` is the symbolic
+name of attribute or the attribute number, the prefix `Tag_RISCV_` can be
+omitted, and second argument is string if attribute number is odd and an integer
+value if the attribute number is even.
+
+Syntax for `.attribute`:
+
+```
+.attribute <NAME_OR_NUMBER>, <ATTRIBUTE_VALUE>
+
+NAME_OR_NUMBER := <attribute-name>
+                | [1-9][0-9]*
+
+ATTRIBUTE_VALUE := <string>
+                 | <number>
+
+```
+
+Attribute name                | Number | Value type     | Description
+:---------------------------- | :----- | :------------- | :----------
+Tag_RISCV_stack_align         |      4 | uleb128        | Indicates the stack alignment requirement in bytes.
+Tag_RISCV_arch                |      5 | NTBS           | Indicates the target architecture of this object.
+Tag_RISCV_unaligned_access    |      6 | uleb128        | Indicates whether to impose unaligned memory accesses in code generation.
+Tag_RISCV_priv_spec           |      8 | uleb128        | Indicates the major version of the privileged specification.
+Tag_RISCV_priv_spec_minor     |     10 | uleb128        | Indicates the minor version of the privileged specification.
+Tag_RISCV_priv_spec_revision  |     12 | uleb128        | Indicates the revision version of the privileged specification.
```

Comment:
Maybe the descriptions could link to the appropriate psABI document sections, for more details?

Reply:
Good suggestion, I think that could prevent out of sync between psABI spec.